src/plugins/lua/antivirus.lua

```diff
@@ -23,46 +23,6 @@ local redis_params
 
 local N = "antivirus"
 
-if confighelp then
```

Comment:
That is what I tried to tell you in IRC ;-) You must please cherry-pick :-) The 1.5.7 release does not know about confighelp. That came later.